modules/nw-route-specific-annotations.adoc

```diff
@@ -70,6 +70,27 @@ This value is applicable to re-encrypt and edge routes only. For more informatio
 Environment variables cannot be edited.
 ====
 
+.Router timeout variables
+
+TimeUnits are represented by a number followed by the unit: us *(microseconds), ms (milliseconds, default), s (seconds), m (minutes), h *(hours), d (days).
+
+The regular expression is: [1-9][0-9]*(us\|ms\|s\|m\|h\|d)
+[cols="3*"]
+|===
+| `ROUTER_BACKEND_CHECK_INTERVAL` | 5000ms | Length of time between subsequent liveness checks on backends.
+| `ROUTER_CLIENT_FIN_TIMEOUT` | 1s | Controls the TCP FIN timeout period for the client connecting to the route. If the FIN sent to close the connection is not answered within the given time, HAProxy will close the connection. This is harmless if set to a low value and uses fewer resources on the router.
```

Comment:
```suggestion
| `ROUTER_CLIENT_FIN_TIMEOUT` | 1s | Controls the TCP FIN timeout period for the client connecting to the route. If the FIN sent to close the connection is not answered within the given time, HAProxy closes the connection. This is harmless if set to a low value and uses fewer resources on the router.
```

> Write in the simple present tense as much as possible. 

I also suggest extending this guidance about lists to the descriptions in this table.

> Use list items that are grammatically parallel.

Reply:
acked